packages/components/src/components/pagination/pagination.tsx

```diff
@@ -58,8 +58,10 @@ export class Pagination {
   @Prop() totalElements?: number = 1;
   /** (optional) Injected styles */
   @Prop() styles?: string;
-  /** (optional) small  */
-  @Prop() small = false;
+  /** DEPRECATED - size should replace small */
```

Comment:
i can maybe cook up something for the deprecation notice, so you can pass in the props that are to be deprecated and when the value isn't nullish you get a warning in the console?
(or you can do it, depends on how much time you have 🙂)